### ISSUE TITLE
Distillation tower shows incomplete structure

### ISSUE THREAD

**Issue by ChrisJStone** [GTNewHorizons/GT-New-Horizons-Modpack#14096]
### Your Pack Version

2.3.5

### Your Server

SP

### Java Version

Java 20

### Type of Server

Single Player

### Your Expectation

Tried to build 4 output distillation tower to process oil. When accessing controller gui shows incomplete structure.

### The Reality

Unable to use distillation tower

### Your Proposal

Fix the distillation tower

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by Bombheads** [comment]
i have the same issue as above, but this also applies to multi block turbine even when built with the multiblock structure hologram project in sneak mode.  Will tell the the structure is finished, but the controller block says its incomplete.

**Comment by ChrisJStone** [comment]
I found removing the top layer of stainless steel casings and filling in the center of the last output layer resolved my issue. I'm not sure if it was ever the case or not. However I thought a final 3x3 layer with no output hatches was required before the distillation tower was complete.

**Comment by chochem** [comment]
no you need an output on the toplevel too

**Comment by Ghezra** [comment]
Right, This is reflected in the preview image:
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/1056989/e659dcca-ee85-4522-b07c-9d5d14ab85f3)
As well as the block description on the controller
![image](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/assets/1056989/62f7ed3a-e941-45f8-9b8f-b0c77f3d1d82)

**Comment by Bombheads** [comment]
> I found removing the top layer of stainless steel casings and filling in the center of the last output layer resolved my issue. I'm not sure if it was ever the case or not. However I thought a final 3x3 layer with no output hatches was required before the distillation tower was complete.

yes this worked, got to try and work out what im doing wrong with the multiblock turbines now.
